src/main/groovy/org/shipkit/internal/gradle/GitPlugin.java

```diff
@@ -27,13 +27,17 @@
  *     <li>gitPush</li>
  *     <li>performGitPush</li>
  *
- *     <li>gitCommitCleanUp</li>
+ *     <li>performGitCommitCleanUp</li>
```

Comment:
Might be good to start documenting the tasks a bit more, writing why we need this task and what it does. For example: "cleans up commit operation so that we can test release conveniently by cleaning up the state of the working copy after the test", "this task will delegate to git soft reset task and then git stash...". Something like that :) This Javadoc will be our public API documentation soon! 1.0 is coming :D